vectordatasource/util.py

```diff
@@ -75,3 +75,21 @@ def calculate_path_major_route(fid):
 def cycling_network(tags, fid):
     # TODO: implement me! current implementation is a stub.
     return None
+
+
+def tag_yes_to_bool(str_or_none):
+    return True if str_or_none == 'yes' else None
```

Comment:
Do we want to also include `true` in that? Sometimes `true` is used as a synonym for `yes` in OSM tags, e.g: `return True if str_or_none in ('yes', 'true') else None`?